src/components/pages/AdminTools/ManageUsers.js

```diff
@@ -117,7 +122,7 @@ const ManageUsersPage = props => {
       first_name: formValues.first_name.trim(),
       last_name: formValues.last_name.trim(),
       email: formValues.email.trim(),
-      role: formValues.role.trim(),
+      role_id: formValues.role_id,

```

Comment:
Updates look great! I'm assuming we're not trimming the role_id because it's not user input. Otherwise, extra sanitation never hurts!